### ISSUE TITLE
GroupedList - inappropriate `aria-label` and `role` attributes on an empty Group

### ISSUE THREAD

**Issue by kroudams** [microsoft/fluentui#21456]
I've got the following errors in 
"Accessibility Insights":

---------------------------------------------------------------------------------------------------------------

Path: `#GroupedListSection30`

Snippet: `<div aria-label="Next processing step" id="GroupedListSection30" class="ms-List"><div role="presentation" class="ms-List-surface"></div></div>`

Fix the following: ARIA attribute cannot be used, add a `role` attribute or use a different element: `aria-label`

=================================================================

Path: `div[aria-labelledby="GroupHeader31"]`

Snippet: `<div class="ms-GroupHeader root-148" role="row" aria-setsize="2" aria-posinset="2" data-is-focusable="true" aria-labelledby="GroupHeader31" aria-expanded="true" aria-level="1" tabindex="0">`

Fix the following:
Required ARIA parents `role` not present: `grid`, `rowgroup`, `table`, `treegrid`

--------------------------------------------------------------------------------------------------------------- 

![GroupedList Aria Errors](https://user-images.githubusercontent.com/87769044/151179997-5be37369-efde-4930-b74a-0610a313f43f.png)

One of the groups in my `GroupedList` is empty.

I can't prevent `GroupedList` from adding `aria-label` on `GroupedListSection` without appropriate `role` and from adding an empty row if the group is empty?

Please note that I need `showEmptyGroups` to be `true`.

Thank you.

**Comment by gouttierre** [comment]
Thanks for filing this, but you’ll need to add a CodePen repro before we can help. You can start from one of the [GroupedList examples](https://developer.microsoft.com/en-us/fluentui#/controls/web/GroupedList) (click “Export to CodePen”) or https://aka.ms/fluentpen.
